nautobot_floor_plan/forms.py

```diff
@@ -42,6 +43,12 @@ class Meta:
             "tags",
         ]
 
+    def __init__(self, *args, **kwargs):
+        """Overwrite the constructor to set initial values for select widget."""
+        super().__init__(*args, **kwargs)
+        self.initial["x_axis_labels"] = get_app_settings_or_config("nautobot_floor_plan", "grid_x_axis_labels")
+        self.initial["y_axis_labels"] = get_app_settings_or_config("nautobot_floor_plan", "grid_y_axis_labels")
+
```

Comment:
`initial=` in the class attributes doesn't seem to work. That's the only solution I found to set the default widget values.